src/main/java/com/dev/cinema/dto/UserResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package com.dev.cinema.dto;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
```

Comment:
I'm not sure, but maybe it is a bad idea to show password to anybody?

Reply:
done